hydraw/app/Main.hs

```diff
@@ -38,6 +40,15 @@ main = do
             putStrLn "Server started..."
             putStrLn $ "Listening on: tcp/" <> show port
         )
+  -- try to parse magic number and if unsuccessfull default to 'Mainnet'
+  -- NOTE: if not set default network is 'Testnet' with magic 42
```

Comment:
Why two different defaults?

Reply:
I thought, ok if the env var is not set we can probably default to testnet 42 but if it is set but network magic is not _parseable_ then let's default to mainnet. Changed now to always default to `Mainnet`.